### DISCUSSION TITLE
Filter hotkeys for specific windows

### DISCUSSION THREAD

**Discussion by obraunsdorf** [rkarp/winapi-easy#5] [Ideas]
Is it possible to only register hotkeys for specific windows? The parameter `HWND` Window's API Function `RegisterHotKeys` (https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerhotkey) seems to implement this functionality. 
How would someone use winapi-easy best to get a HWND to the processes main window and pass it to `RegisterHotKeys`?

Thanks for your help!

**Comment by rkarp** [comment]
It surely should be possible, but not with `RegisterHotkey`. According to the docs, passing a window handle would only mean that the associated window would receive the messages, nothing more. The hotkeys would still be global.
So, implementing this would require other functions from the Windows API, something which I haven't looked into yet.

Constructing / getting a window can already be done with winapi-easy, though I consider the API for this highly experimental still. But you can take a look at the `new_window` and `get_window_infos` examples.
